commit fa6870869e5d2ad90251027f23901825a34c971b
Author: Cole Robinson <crobinso@redhat.com>
Date:   Wed Feb 8 07:24:58 2012 -0500

    host: Don't show error for all interfaces if one is busted

diff --git a/src/virtManager/host.py b/src/virtManager/host.py
index bdbd124..7a8a7f3 100644
--- a/src/virtManager/host.py
+++ b/src/virtManager/host.py
@@ -1072,7 +1072,12 @@ class vmmHost(vmmGObjectUI):
                 break
             idx += 1
 
-        used_by = util.iface_in_use_by(self.conn, name)
+        # This can fail if other interfaces are busted, so ignore errors
+        used_by = None
+        try:
+            used_by = util.iface_in_use_by(self.conn, name)
+        except Exception, e:
+            logging.debug("Error looking up iface usage: %s", e)
         self.widget("interface-inuseby").set_text(used_by or "-")
 
         # IP info
